Source/WebKit/Shared/Cocoa/WebIconUtilities.mm

```diff
@@ -116,6 +116,8 @@ PlatformImagePtr fallbackIconForFile(NSURL *file)
 #endif
 }
 
+const CFStringRef kCGImageSourceEnableRestrictedDecoding = CFSTR("kCGImageSourceEnableRestrictedDecoding");
```

Comment:
Not new to your patch, and should be done separately, but we should make better use of `pal/spi/cg/ImageIOSPI.h`. `ImageDecoderCG` also defines a bunch of these.